### PR TITLE
 Fix WebSocket integration test files

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
Fixes files related to WebSocket integration tests

### CODE REVIEW

File: tests/ballerina-integration-test/src/test/resources/websocket/wsservices/19_client_service.bal [hunk 1]
```diff
@@ -45,14 +45,14 @@ service<http:WebSocketService> clientFailure200 bind { port: 9200 } {
...
         };
     }
 }
-service<http:WebSocketService> callback200 {
-}
-service<http:WebSocketClientService> ClientService200 {
-}
+service callback200 = @http:WebSocketServiceConfig {} service {
+};
+service<http:WebSocketClientService> ClientService200 = @http:WebSocketServiceConfig {} service{
```
Comment (MaryamZi) [inline tests/ballerina-integration-test/src/test/resources/websocket/wsservices/19_client_service.bal]: shouldn't `<http:WebSocketClientService>` here be removed?

File: tests/ballerina-integration-test/src/test/resources/websocket/wsservices/07_pushText_failure.bal [hunk 2]
```diff
@@ -17,8 +17,8 @@
 import ballerina/http;
 import ballerina/io;
 
-service<http:WebSocketService> pushTextFailure bind { port: 9089 } {
-    onOpen(endpoint caller) {
+service pushTextFailure on new http:WebSocketListener(9089) {
+    resource function onOpen(endpoint caller) {
```
Comment (MaryamZi) [inline tests/ballerina-integration-test/src/test/resources/websocket/wsservices/07_pushText_failure.bal]: `endpoint` needs to change?

File: tests/ballerina-integration-test/src/test/resources/websocket/wsservices/19_client_service.bal [hunk 3]
```diff
@@ -34,7 +34,7 @@ service<http:WebSocketService> clientFailure200 bind { port: 9200 } {
         }
     }
 
-    onText(endpoint caller, string text) {
+    resource function onText(WebSocketCaller caller, string text) {
```
Comment (MaryamZi) [inline tests/ballerina-integration-test/src/test/resources/websocket/wsservices/19_client_service.bal]: Shouldn't this be `http:WebSocketCaller`?

File: tests/ballerina-integration-test/src/test/resources/websocket/wsservices/17_simple_proxy_server.bal [hunk 4]
```diff
@@ -65,31 +65,31 @@ service<http:WebSocketService> simpleProxy9 bind { port: 9099 } {
...
+    resource function onText(http:WebSocketCaller wsEp, string text) {
         endpoint http:WebSocketListener serviceEp = getAssociatedListener3(wsEp);
         var returnVal = serviceEp->pushText(text);
         if (returnVal is error) {
              panic returnVal;
         }
     }
 
-    onBinary(endpoint wsEp, byte[] data) {
+    resource function onBinary(endpoint wsEp, byte[] data) {
```
Comment (Manuri) [inline tests/ballerina-integration-test/src/test/resources/websocket/wsservices/17_simple_proxy_server.bal]: Need to replace `endpoint` with the relevant type. Please check other places.

File: tests/ballerina-integration-test/src/test/resources/websocket/wsservices/19_client_service.bal [hunk 5]
```diff
@@ -34,7 +34,7 @@ service<http:WebSocketService> clientFailure200 bind { port: 9200 } {
...
         endpoint http:WebSocketClient wsClientEp {
```
Comment (wggihan) [inline tests/ballerina-integration-test/src/test/resources/websocket/wsservices/19_client_service.bal]: What is this endpoint usage?

File: tests/ballerina-integration-test/src/test/resources/websocket/wsservices/01_isOpen.bal [hunk 6]
```diff
@@ -17,26 +17,26 @@
...
-http:WebSocketListener listener = new;
+listener http:WebSocketListener socketListener = new(9078);
 
 @http:WebSocketServiceConfig {
     path: "/"
 }
-service<http:WebSocketService> isOpen bind { port: 9078 } {
-    onOpen(endpoint caller){
-        listener = untaint caller;
+service isOpen on socketListener{
```
Comment (Bhashinee) [inline tests/ballerina-integration-test/src/test/resources/websocket/wsservices/01_isOpen.bal]: Let's add a space after 'socketListener' -> service isOpen on socketListener {

**Pull request merged**